### PR TITLE
standardization: Create a python package of zircbot

### PR DESCRIPTION
**Pull request by Spredzy**
Currently zircbot is a python script. It does one thing and does it
well, transform Trello event into IRC message.

We can imagine that Trello is just a backend and that we might want to
implement others with the notion of plugins/drivers.

In order to add more feature to this module applying some python
packaging rules from the beginning might be a good idea

### CODE REVIEW

**Comment by Spredzy** [comment]
DO NO MERGE yet - just to gather feedbacks

File: publish.py [hunk 1]
```diff
@@ -45,9 +47,10 @@ def main():
     s = ZmqPubConnection(zf, e)
 
     def publish():
-        data = str(time.time())
-        print "publishing %r" % data
-        s.publish(data)
+        #data = str(time.time())
+        data = json.load(open('/tmp/sensu.json', 'r'))
```
Comment (fredericlepied) [inline publish.py]: could we read the filename on the command line to make it more generic?

File: zircbot/__init__.py [hunk 2]
```diff
@@ -33,6 +33,8 @@
 from twisted.internet import reactor, protocol
 from twisted.python import log
 from twisted.words.protocols import irc
+from zircbot.backends import trello
```
Comment (fredericlepied) [inline zircbot/__init__.py]: I would prefer plugins rather than backends. As there is no database or whatever backend involved here.

Comment (Spredzy) [inline zircbot/__init__.py]: Done.

File: publish.py (hunk 1 above)
Comment (Spredzy) [inline publish.py]: Done.

**Comment by fredericlepied** [comment]
thx Yanis!

**Pull request merged**